lib/room-app-stack.ts

```diff
@@ -40,8 +43,8 @@ export class RoomAppStack extends cdk.Stack {
 
     const connectFn = new lambda.Function(this, "ConnectionHandler", {
       runtime: lambda.Runtime.NODEJS_14_X,
-      handler: "index.connect",
-      code: lambda.Code.fromAsset("backend/connect"),
+      handler: "connections.connectHandler",
```

Comment:
@douglasnaphas None of these new handlers seem to be being called

```
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
|   timestamp   |                                                                                                                                                                                                                                                                                                                                                                                                                                message                                                                                                                                                                                                                                                                                                                                                                                                                                |
|---------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| 1624049350418 | START RequestId: 0e871638-9fe2-4f5c-aecc-df7d7210522f Version: $LATEST                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
| 1624049350420 | 2021-06-18T20:49:10.420Z 0e871638-9fe2-4f5c-aecc-df7d7210522f INFO defaulted                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
| 1624049350420 | 2021-06-18T20:49:10.420Z 0e871638-9fe2-4f5c-aecc-df7d7210522f INFO invoked!                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
| 1624049350421 | 2021-06-18T20:49:10.420Z 0e871638-9fe2-4f5c-aecc-df7d7210522f INFO event:                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
| 1624049350423 | 2021-06-18T20:49:10.423Z 0e871638-9fe2-4f5c-aecc-df7d7210522f INFO {   requestContext: {     routeKey: '$default',     messageId: 'BI3-8cVlIAMCExQ=',     eventType: 'MESSAGE',     extendedRequestId: 'BI3-8GDQoAMFSeQ=',     requestTime: '18/Jun/2021:20:49:09 +0000',     messageDirection: 'IN',     stage: 'ws',     connectedAt: 1624049349554,     requestTimeEpoch: 1624049349895,     identity: { userAgent: 'Amazon CloudFront', sourceIp: '130.176.163.85' },     requestId: 'BI3-8GDQoAMFSeQ=',     domainName: 'c92foxzd3i.execute-api.us-east-1.amazonaws.com',     connectionId: 'BI3-5cVXIAMCExQ=',     apiId: 'c92foxzd3i'   },   body: 'Hello WS, I have connected.',   isBase64Encoded: false }                                                                                                                                                   |
| 1624049350423 | 2021-06-18T20:49:10.423Z 0e871638-9fe2-4f5c-aecc-df7d7210522f INFO context:                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
| 1624049350423 | 2021-06-18T20:49:10.423Z 0e871638-9fe2-4f5c-aecc-df7d7210522f INFO {   callbackWaitsForEmptyEventLoop: [Getter/Setter],   succeed: [Function (anonymous)],   fail: [Function (anonymous)],   done: [Function (anonymous)],   functionVersion: '$LATEST',   functionName: 'Dan-mullerRoom-appInitial-c-DefaultHandler604DF7AC-lh8eN1VgBkrU',   memoryLimitInMB: '3000',   logGroupName: '/aws/lambda/Dan-mullerRoom-appInitial-c-DefaultHandler604DF7AC-lh8eN1VgBkrU',   logStreamName: '2021/06/18/[$LATEST]cf352e69a92042b2a9a9503b24689688',   clientContext: undefined,   identity: undefined,   invokedFunctionArn: 'arn:aws:lambda:us-east-1:369840494493:function:Dan-mullerRoom-appInitial-c-DefaultHandler604DF7AC-lh8eN1VgBkrU',   awsRequestId: '0e871638-9fe2-4f5c-aecc-df7d7210522f',   getRemainingTimeInMillis: [Function: getRemainingTimeInMillis] }  |
| 1624049350424 | END RequestId: 0e871638-9fe2-4f5c-aecc-df7d7210522f                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
| 1624049350425 | REPORT RequestId: 0e871638-9fe2-4f5c-aecc-df7d7210522f Duration: 5.95 ms Billed Duration: 6 ms Memory Size: 3000 MB Max Memory Used: 65 MB Init Duration: 152.04 ms                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

These logs look like it was produced by the pre-PR version of the code